apps/web/src/pages/Pools/AssessmentPlanBuilderPage/components/AssessmentDetailsDialog.tsx

```diff
@@ -392,6 +441,27 @@ const AssessmentDetailsDialog = ({
     createAssessmentStepFetching ||
     createOrUpdateScreeningQuestionAssessmentStepMutationFetching;
 
+  const missingSkills = allPoolSkills.filter(({ assessmentSteps }) => {
+    const steps = unpackMaybes(assessmentSteps);
+
+    // Behavioral skill attached to no assessment steps
+    if (steps.length === 0) {
+      return true;
+    }
+
+    // Skill is only attached to Application screening. All technical skills are attached by default.
+    if (
+      steps.length === 1 &&
+      steps?.find(
+        (step) => step?.type === AssessmentStepType.ApplicationScreening,
+      )
+    ) {
+      return true;
+    }
```

Comment:
I'm wonder if this would work as just something like this:

```ts
if(
    steps.length === 0 || 
    (steps.length === 1 && steps[0].type === AssessmentStepType.ApplicationScreening)
 ) {
    return true;
}
```

Could reduce the need for two if statements with the same result and since we already check the steps length we shouldn't need to find and can access it directly from the 0th index.

Reply:
Nice suggestion, it worked perfectly. However, it turns out we don't want to add any pool skills only attached to applicant screening (all technical skills by default). There's plans of a future design to improve this overall. So, for now just leave it as any pool skill without at least 1 assessment, including application screening.

Changed in [aa2e90f](https://github.com/GCTC-NTGC/gc-digital-talent/pull/9993/commits/aa2e90f1fd988e595902e113739a25f420453d9e)